package.json

```diff
@@ -51,17 +49,17 @@
     "moment": "^2.24.0",
     "noop-ts": "^1.0.3",
     "normalize.css": "^8.0.1",
-    "react": "^16.9.6",
+    "react": "^17.0.1",
     "react-countdown": "^2.3.2",
     "react-dom": "npm:@hot-loader/react-dom@^17.0.1",
     "react-i18next": "^11.16.5",
     "react-query": "^3.34.16",
     "react-router": "^5.2.1",
     "react-router-dom": "^5.0.1",
     "react-scripts": "4.0.3",
-    "react-toastify": "^9.0.1",
+    "react-toastify": "9.0.1",
     "react-uid": "^2.3.1",
-    "recharts": "^2.1.9",
+    "recharts": "2.1.9",
```

Comment:
Why do we need to fix the versions of these packages?

Reply:
Recharts was throwing an error relating to a resizing property missing from window. react-toastify had a esm compilation issue if I recall correctly